clients/privacy-center/cypress/e2e/banner-overlay-dismissal.cy.ts

```diff
@@ -64,24 +62,20 @@ describe("Banner and overlay dismissal", () => {
         assertDismissCalled();
       });
 
-      it("Should dismiss the banner by clicking outside", () => {
+      it("Should not dismiss the banner by clicking outside", () => {
```

Comment:
nit: this test is named incorrectly - it should be:
```suggestion
      it("Should not dismiss the modal by clicking outside", () => {
```

..and subsequently, the assertions aren't right.

Reply:
Let's fix that in a followup